tests/countries/test_syrian_arab_republic.py

```
@@ -0,0 +1,147 @@
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.syrian_arab_republic import SyrianArabRepublic, SY, SYR
+from tests.common import CommonCountryTests
+
+
+class TestSyrianArabRepublic(CommonCountryTests, TestCase):
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)